types/index.d.ts

```diff
@@ -39,328 +38,335 @@ declare module nlp {
-    some(fn: Function): Document<World>
+    some(fn: Function): Document<W>
     /**  sample a subset of the results */
-    random(n?: number): Document<World>
+    random(n?: number): Document<W>
 
     // Insert
     /**  substitute-in new content */
-    replaceWith(text: string | Function, keepTags?: boolean | object, keepCase?: boolean): Document<World>
+    replaceWith(text: string | Function, keepTags?: boolean | object, keepCase?: boolean): Document<W>
     /**  search and replace match with new content */
-    replace(match: string, text?: string | Function, keepTags?: boolean | object, keepCase?: boolean): Document<World>
+    replace(match: string, text?: string | Function, keepTags?: boolean | object, keepCase?: boolean): Document<W>
     /**  fully remove these terms from the document */
-    delete(match: string): Document<World>
+    delete(match: string): Document<W>
     /**  add these new terms to the end (insertAfter) */
-    append(text: string): Document<World>
+    append(text: string): Document<W>
     /**  add these new terms to the front (insertBefore) */
-    prepend(text: string): Document<World>
+    prepend(text: string): Document<W>
     /**  add these new things to the end */
-    concat(text: string): Document<World>
+    concat(text: string): Document<W>
 
     // transform
     /**re-arrange the order of the matches (in place) */
-    sort(method?: string | Function): Document<World>
+    sort(method?: string | Function): Document<W>
     /**reverse the order of the matches, but not the words */
-    reverse(): Document<World>
+    reverse(): Document<W>
     /** clean-up the document, in various ways */
     normalize(options?: string | object): string
     /** remove any duplicate matches */
-    unique(): Document<World>
+    unique(): Document<W>
     /**  return a Document with three parts for every match ('splitOn') */
-    split(match?: string): Document<World>
+    split(match?: string): Document<W>
     /**  separate everything after the match as a new phrase */
-    splitBefore(match?: string): Document<World>
+    splitBefore(match?: string): Document<W>
     /**  separate everything before the word, as a new phrase */
-    splitAfter(match?: string): Document<World>
+    splitAfter(match?: string): Document<W>
     /** split a document into labeled sections  */
-    segment(regs: object, options?: object): Document<World>
+    segment(regs: object, options?: object): Document<W>
     /** make all phrases into one phrase  */
-    join(str?: string): Document<World>
+    join(str?: string): Document<W>
 
     // Output
     /**  return the document as text */
     text(options?: string | object): string
     /**  pull out desired metadata from the document */
     json(options?: string | object): any
     /** some named output formats */
-    out(format?: string): string
+    out(format?: 'grid' | 'color' | 'normal' | 'csv' | 'newlines' | 'html' | 'text'): string
```

Comment:
@spencermountain Would you mind double checking these? I did it awhile ago...

Reply:
Sure, I'll update this on dev locally.